src/metabase/sync/sync_metadata/sync_table_privileges.clj

```diff
@@ -15,8 +15,7 @@
   [database :- (mi/InstanceOf :model/Database)]
   (let [driver (driver.u/database->driver database)]
     (when (and (not= :redshift driver)
```

Comment:
You can remove the redshift exclusion now and the comment below, so this is just
```clojure
    (when (driver/database-supports? driver :table-privileges database) ...)
```

Reply:
I intentionally updated the comment and left this so we didn't unintentionally enabled it again if we roll the other changes forward again.